src/coreComponents/physicsSolvers/fluidFlow/wells/WellControls.cpp

```diff
@@ -65,7 +65,7 @@ WellControls::WellControls( string const & name, Group * const parent )
   registerWrapper( viewKeyStruct::targetPhaseRateString(), &m_targetPhaseRate ).
     setDefaultValue( 0.0 ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "Target phase volumetric rate" );
+    setDescription( "Target phase volumetric rate [sm^3/s]" );
```

Comment:
Is this intended? `[sm^3/s]` -> `[m^3]`?

Reply:
Yes, this can be confusing, [sm^3/s] means "surface m^3 per second", it is now written explicitly in the code